neatnet/simplify.py

```diff
@@ -689,13 +689,21 @@ def neatify(
     For further information on face artifact detection and extraction
     see :cite:`fleischmann_shape-based_2024`.
 
+    This algorithm is designed for use with only "street" network geometries as input.
+    While passing in other types of pathing (e.g., sidewalks, canals) will likely yield
+    valid geometric results, those results will likely not represent the morphology
+    appropriately.
```

Comment:
That is a question. It may as well work but it very much depends on the behaviour of artifact detection. I would say something like "behaviour is untested" rather that straight say it is probably wrong.